.github/workflows/label-last-commenter.yml

```diff
@@ -0,0 +1,31 @@
+name: 'Tag issues with last commenter'
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    if: ${{ !github.event.issue.pull_request }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Add label if commenter is not member
+        if: |
+          github.event.comment.author_association != 'COLLABORATOR'
+          && github.event.comment.author_association != 'MEMBER'
+          && github.event.comment.author_association != 'OWNER'
+        uses: actions-ecosystem/action-add-labels@v1
+        with:
+          labels: 'Status: User responded'
```

Comment:
Can we use "Waiting for: Team"?

Then we can eventually rename "Status: Needs more information" to "Waiting for: User".

Reply:
I like that

Reply:
changed it! 🚀